app-antivirus/clamtk/metadata.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<upstream>
+		<bugs-to>https://github.com/dave-theunsub/clamtk/issues</bugs-to>
+		<remote-id type="github">dave-theunsub/clamtk</remote-id>
+	</upstream>
```

Comment:
please move `upstream` section to the bottom.
according to this order, left to right maps top to bottom.
```sh
maintainer|longdescription|slots|stabilize-allarches|use|upstream
```

Reply:
Thank you, done!